docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,205 @@
+
+### Review readers
+
+Review readers access reviews to make informed decisions on selecting services. Readers rate reviews by casting up or down votes. The review readers' ratings are factored into the algorithm that rates the reviews, with the authors of the highest-rated reviews getting awarded. Although it is not implemented as part of this guide, a proportion of the review reward may be awarded to review readers who has participated in rating reviews by casting their votes.
+
+### Moderators
+
+Moderators monitor content of the reviews and can delete any reviews that contain inappropriate content. Moderators also keep an eye on the number of reviews registered for a service to ensure that they do not grow indefinitely.
+
+The incentive mechanism is not implemented for this guide, but service owners can all pay into a pool that goes to moderators, on a rolling basis. People can stake moderators to influence what proportion of the reward each moderator gets, up to a limit (imagine how validators are staked on-chain), and moderator decisions are decided by quorum of stake weight. This is a way to ensure that moderators have incentives to do their job well.
+
+## How reviews are rated
+
+The reviews are rated (scored) on-chain using following criteria
+- Intrinsic score (IS)
+  - Length of content
+- Extrinsic score (ES)
+  - Number of votes received
+- Verification multiplier (VM)
+  - Reviews with PoE get a multiplier to improve rating
+
+```
+Total Score = (IS + ES) * VM
+```
+
+## Smart contracts
+
+There are several modules that create the backend logic for the example.
+
+### dashboard.move
+
+The `dashboard.move` module defines the `Dashboard` struct that is used to group services.
+
+```move
+struct Dashboard has key, store {
+  id: UID,
+  service_type: String
+}
+```
+
+The services are grouped by attributes, which can be cuisine type, geographical location, operating hours, google maps ID, and so on. To keep it basic, the example stores only `service_type` (for example, fast food, Chinese, Italian).
+
+### service.move
+
+This module defines the `Service` struct, which service owners manage.
+
+```move
+struct Service has key, store {
+  id: UID,
+  reward_pool: Balance<SUI>,
+  reward: u64,
+  top_reviews: MultiMap<ID>,
+  reviews: Table<ID, ID>,
+  moderators: Table<address, address>,
+  overall_rate: u64,
+  name: String
+}
+```
+
+All the submitted reviews are stored in `reviews` field, and top rated reviews are stored in `top_reviews` field, which has `MultiMap<ID>` type. The [multimap module](#multimap) defines the type.
+
+The `reward_pool` and `reward` fields store the available reward information.
+
+```move
+struct AdminCap has key, store {
+  id: UID,
+  service_id: ID
+}
+struct Moderator has key {
+  id: UID,
+  service_id: ID,
+}
+```
+`SERVICE OWNERS` are given `AdminCap`, and `MODERATORS` are given `Moderator` to allow them to perform privileged operations.
```

Comment:
Let's link to documentation on "capabilities" here and make it clear that this is an example of the capability pattern.

Reply:
I added a link to "capabilities" documentation as suggested.